### PR TITLE
Remove source build patches

### PR DESCRIPTION
**Pull request by eerhardt**
FYI - @weshaggard

### CODE REVIEW

**Comment by eerhardt** [comment]
test Windows_NT arm Debug Build

**Comment by eerhardt** [comment]
ARM builds are currently broken by https://github.com/dotnet/core-eng/issues/1835

**Comment by eerhardt** [comment]
Merging into `dev/release/2.0.0`

File: dir.props [hunk 1]
```diff
@@ -88,18 +101,6 @@
     <HostPolicyVersion Condition="'$(UseShippedHostPolicyPackage)' == 'true'">2.0.0</HostPolicyVersion>
   </PropertyGroup>
 
-  <!-- list of nuget package sources passed to dnu -->
-  <ItemGroup>
-    <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
-    <DotnetSourceList Include="$(OverridePackageSource)" />
```
Comment (weshaggard) [inline dir.props]: Don't you still need a replacement for the DotnetSourceList somewhere?  I seem to be missing the connection between the RestoreSources property above, the DotnetSourceLIst here and the use of DotnetRestoreSource below.

Comment (weshaggard) [reply dir.props]: Nevermind I found the missing link. https://github.com/dotnet/buildtools/pull/1707

Comment (eerhardt) [reply dir.props]: The advantage with this approach is that I can now open https://github.com/dotnet/core-setup/blob/master/src/managed/Microsoft.Extensions.DependencyModel/Microsoft.Extensions.DependencyModel.csproj in VS 2017, and restore works correctly.

**Pull request merged**